apis/r/tests/testthat/test-example-datasets.R

```diff
@@ -1,7 +1,7 @@
 test_that("example dataset access", {
   expect_length(
     list_datasets(),
-    length(dir(example_data_dir()))
+    length(dir(example_data_dir())) - 1 # There is a README.md
```

Comment:
I don't see a place in `README.md` that explains why this `- 1` exists

Reply:
The `README.md` file is an item within `example_data_dir`.

Reply:
I can say that in the `README.md` file.

Reply:
@mojaveazure https://github.com/single-cell-data/TileDB-SOMA/pull/3303/commits/b6c59b4c72142a6fd01f4110e08cee0ccf9a45b8